Index: cga-backup/client/cgabackup
===================================================================
--- cga-backup.orig/client/cgabackup	2010-07-23 22:40:12.130355890 +0200
+++ cga-backup/client/cgabackup	2010-07-23 22:41:21.100352829 +0200
@@ -41,7 +41,7 @@
 $globalconf{rsh}="rsh -lroot";
 $RSYNC="rsync";
 $SENDMAIL="/usr/lib/sendmail";
-$CONFIGFILE="/etc/cgabackup.conf";
+$CONFIGFILE="/etc/cga-backup/client.conf";
 $VERBOSE=0;
 
 # Hostname eruieren
